tests/rptest/perf/small_batches_test.py

```diff
@@ -19,31 +19,44 @@ def omb_test(self):
         workload = {
             "name": "SmallBatchesWorkload",
             "topics": 1,
-            "partitions_per_topic": 20000,
+            "partitions_per_topic": 100,
             "subscriptions_per_topic": 1,
-            "consumer_per_subscription": 200,
-            "producers_per_topic": 200,
-            "producer_rate": 150_000,
+            "consumer_per_subscription": 1,
+            "producers_per_topic": 10,
+            "producer_rate": 30_000,
             "message_size": 1024,
             "payload_file": "payload/payload-1Kb.data",
             "consumer_backlog_size_GB": 0,
             "test_duration_minutes": 5,
             "warmup_duration_minutes": 1,
         }
+        driver = {
+            "name": "SmallBatchesDriver",
+            "replication_factor": 3,
+            "request_timeout": 300000,
+            "producer_config": {
+                "enable.idempotence": "false",
+                "acks": "all",
+                "linger.ms": 1,
+                "max.in.flight.requests.per.connection": 10,
```

Comment:
I think we should really test we idempotence on as it's the more common codepath.

Reply:
Fair enough, switching it over.